baselines/queries/hybrid.json

```diff
@@ -34,7 +34,7 @@
         },
         {
             "name": "offset_limit_rrf_unfiltered",
-            "query": "SELECT c.index, c.title FROM c ORDER BY RANK RRF(FullTextScore(c.title, 'John'), FullTextScore(c.text, 'United States')) OFFSET 0 LIMIT 13",
```

Comment:
Changing this appears to be necessary in order to get deterministic results out of the .NET SDK. I'm putting together a full repro, but it seems like the .NET SDK can return non-deterministic results because it uses an "unstable" sort when sorting by scores, which means the ordering of items with the same score is undefined and non-deterministic.

Rust and Python both have stable sorts which effectively means ties are broken by sorting in ascending order of RID (because the items are first sorted by RID, then by score). However, our "baselines" are computed from .NET, which has this instability issue.